bin/quickpkg

```diff
@@ -132,14 +148,14 @@ def quickpkg_atom(options, infos, arg, eout):
 				vardb.aux_update(cpv, update_metadata)
 			xpdata = xpak.xpak(dblnk.dbdir)
 			binpkg_tmpfile = os.path.join(bintree.pkgdir,
-				cpv + ".tbz2." + str(os.getpid()))
+				cpv + ".tar." + compression["suffix"] + "." + str(os.getpid()))
 			ensure_dirs(os.path.dirname(binpkg_tmpfile))
 			# The tarfile module will write pax headers holding the
 			# xattrs only if PAX_FORMAT is specified here.
-			tar = tarfile.open(binpkg_tmpfile, "w:bz2",
-				format=tarfile.PAX_FORMAT if xattrs else tarfile.DEFAULT_FORMAT)
-			tar_contents(contents, root, tar, protect=protect, xattrs=xattrs)
-			tar.close()
+			with open(binpkg_tmpfile, "wb") as fp:
+				proc = subprocess.Popen([compression["compress"], "-"], stdin=subprocess.PIPE, stdout=fp, shell=True)
```

Comment:
We could change that to be a tuple of things for try, which will work with varexpand:
```
decompress: ("${PORTAGE_BUNZIP2_COMMAND}", "${PORTAGE_BZIP2_COMMAND} -d")
```